js-console/src/main/scala/org/typelevel/log4cats/console/ConsoleF.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats
+package console
+
+import cats.effect.Sync
+
+class ConsoleF[F[_]: Sync] {
```

Comment:
Actually I don't think we should expose this either. Our goal is not really to safely expose JS APIs, it's to provide a log4cats implementation.

Reply:
Addressed in 3240c588.